www/source/blog/2018-02-21-Habitat-Helm.html.md

```diff
@@ -57,24 +43,29 @@ Once you've cloned core-plans locally, go to `core-plans/nginx` and enter the Ha
 
 * Enter the Habitat Studio within the nginx core plan with `hab studio enter`
 * Build the latest nginx package and stores it as a habitat artifact (“hart”) in the nginx plan’s `results` directory: `build`
-* Export the hab artifact to a Helm chart: `hab pkg export helm results/<yourfilename>.hart` This command will look very similar to `hab pkg export helm results/tdrew-nginx-1.11.10-20180221053714-x86_64-linux.hart`
+* Initialize the Helm client-side: `helm init -c`
+* Export the hab artifact to a Helm chart:
+
+`hab pkg export helm --download-deps results/<yourfilename>.hart`
+
+This command will look very similar to
+
+`hab pkg export helm --download-deps results/tdrew-nginx-1.11.10-20180221053714-x86_64-linux.hart`
+
 * Exit the Habitat Studio: `exit`
 
 You now have a Helm chart for core-plans/nginx in your core-plans/nginx/nginx-<version>-<datestamp>/ directory
```

Comment:
Think this para is now a leftover from before when we were building the operator. Can be removed I think?

Reply:
Good catch!

Reply:
actually no! This is about the helm chart dir that was just generated above.

Reply:
I am sorry. Meant to be placed below this:
>  I suggest you pwd for the above ^^ so you can use it in another tab when you have started your cluster and want to know where your helm chart lives

The talk about needing to use another tab...

Reply:
I'm not sure this is related to that since the instruction didn't actually launch operator and the para isn't even talking of the operator. Am I missing something?